CoreFoundation/Parsing.subproj/CFBinaryPList.c

```diff
@@ -1248,12 +1251,19 @@ CF_PRIVATE bool __CFBinaryPlistCreateObjectFiltered(const uint8_t *databytes, ui
 	    CFSwappedFloat32 swapped32;
 	    memmove(&swapped32, ptr, 4);
 	    float f = CFConvertFloat32SwappedToHost(swapped32);
-	    *plist = CFNumberCreate(allocator, kCFNumberFloat32Type, &f);
-	    // these are always immutable
-	    if (objects && *plist) {
-		CFDictionarySetValue(objects, (const void *)(uintptr_t)startOffset, *plist);
-	    }
-	    return (*plist) ? true : false;
+            if (outPlistTypeID) *outPlistTypeID = _kCFRuntimeIDCFNumber;
```

Comment:
Nit: likewise indentation appears off here and below, and it appears pervasively throughout the rest of the file.